server/storage.lua

```diff
@@ -225,28 +225,12 @@ end
 ---@param citizenId string
 ---@return boolean success if operation is successful.
 function DeletePlayerEntity(citizenId)
-    local playerTables = {
-        'players',
-        'apartments',
-        'bank_accounts',
-        'crypto_transactions',
-        'phone_invoices',
-        'phone_messages',
-        'playerskins',
-        'player_contacts',
-        'player_houses',
-        'player_mails',
-        'player_outfits',
-        'player_vehicles',
-    }
-
-    local query = "DELETE FROM %s WHERE citizenid = ?"
+    local query = "DELETE FROM %s WHERE %s = ?"
```

Comment:
```suggestion
    local query = 'DELETE FROM %s WHERE %s = ?'
```

pretty please thank you